s2wav.egg-info/PKG-INFO

```diff
@@ -0,0 +1,78 @@
+Metadata-Version: 2.1
```

Comment:
So we shouldn't include s2wav.egg folders as they are basically the build package. Just need to run 
` git rm -rf s2wav.egg-info `
 and then push to remove this